### ISSUE TITLE
Package cannot be found

### ISSUE THREAD

**Issue by choopage** [esoyke/ngx-international-phone-number2#3]
![Screenshot 2019-11-11 at 11 32 57 AM](https://user-images.githubusercontent.com/6467136/68559208-1d744580-0477-11ea-9158-eae98c2fe48a.png)

https://stackblitz.com/edit/ngx-international-phone-number2

**Comment by esoyke** [comment]
Odd, I've added to a couple projects with no problem from command line
'npm i ngx-international-phone-number2' pulls it in. 

Perhaps something is missing with how stackblitz imports libraries? Even stranger is when I expand the Dependencies window in your Stackblitz, it shows that it has the latest version of the library.

**Comment by choopage** [comment]
Exactly. I was trying to use and I could reproduce the error through
stackblitz.

Is it something we can fix. Through redeployment?

Don't forget to up to version when redeploy thanks.

**Comment by choopage** [comment]
hi @esoyke  did you manage to re-deploy it?
